### ISSUE TITLE
fix TODO's in code

### ISSUE THREAD

**Issue by RobTillaart** [RobTillaart/AM232X#6]
either fix them or create a real issue.

**Comment by RobTillaart** [comment]
make a separate wakeup function
```cpp
bool wakeup()
{
  // wake up the sensor - see 8.2
  Wire.beginTransmission(AM232X_ADDRESS);
  int rv = Wire.endTransmission();
  delayMicroseconds(1000);          // TODO tune
  return (rv == 0;)
}
```
can be used in read- and write-register and maybe even as user function.

**Comment by RobTillaart** [comment]
idea to improve, no final one

```cpp
  int bytes = Wire.requestFrom(AM232X_ADDRESS, length);

  for (int i = 0; i < bytes; i++)
  {
    bits[i] = Wire.read();
  }
  // ANALYZE ERRORS
  // will not detect if we requested 1 byte as that will
  // return 5 bytes as requested. E.g. getStatus()
  // TODO: design a fix.
  if (bytes != length)
  {
    if (bytes >= 3)
    {
      switch(bits[3])
      {
      case 0x80: return AM232X_ERROR_FUNCTION;
      case 0x81: return AM232X_ERROR_ADDRESS;
      case 0x82: return AM232X_ERROR_REGISTER;
      case 0x83: return AM232X_ERROR_CRC_1;  // prev write had a wrong CRC
      case 0x84: return AM232X_ERROR_WRITE_DISABLED;
      default:   return AM232X_ERROR_UNKNOWN;
      }
    }
   else return AM232X_ERROR_TO_FEW_BYTES_REQUESTFROM;   // or so...

```

**Comment by RobTillaart** [comment]
fixed in 0.3.1
